spec/views/student/show_spec.rb

```diff
@@ -1,42 +1,41 @@
 require 'rails_helper'
 
-RSpec.describe 'coordinators/show.html.erb', type: :view do
+RSpec.describe 'students/show.html.erb', type: :view do
```

Comment:
Did you pend these tests until we resolve how we related tutors and coordinators?